File: Base/src/main/java/com/lalifa/ext/Account.java
package com.lalifa.ext;

import android.text.TextUtils;

import java.io.Serializable;
import java.util.Objects;

public class Account implements Serializable {
    String userId;
    String userName;
    String avatar;
    String imToken;

    public Account(String userId, String userName, String avatar, String imToken) {
        this.userId = userId;
        this.userName = userName;
        this.avatar = avatar;
        this.imToken = imToken;
    }

    public String getUserId() {
        return userId;
    }

    public String getUserName() {
        return userName;
    }

    public String getAvatar() {
        return TextUtils.isEmpty(avatar) ? "" : Config.FILE_PATH + avatar;
    }

    public String getImToken() {
        return imToken;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        Account accout = (Account) o;
        return Objects.equals(userId, accout.userId);
    }

    @Override
    public int hashCode() {
        return Objects.hash(userId);
    }
}

File: Base/src/main/java/com/lalifa/ext/Account.kt
package com.lalifa.ext

import android.text.TextUtils
import java.io.Serializable
import java.util.*

data class Account(
    var userId: String = "",
    var userName: String = "",
    var avatar: String = "",
    var imToken: String = "",
    //头像框
    var frame: String = "",
    //座驾
    var car: String = "",
    //气泡
    var bubble: String = "",
    //音波
    var sound: String = "",
) : Serializable {
    @JvmName("getAvatar1")
    fun getAvatar(): String {
        return if (TextUtils.isEmpty(avatar)) "" else Config.FILE_PATH + avatar
    }

    @JvmName("setAvatar1")
    fun setAvatar(avatar: String) {
        this.avatar = avatar
    }

    override fun equals(o: Any?): Boolean {
        if (this === o) return true
        if (o == null || javaClass != o.javaClass) return false
        val accout = o as Account
        return userId == accout.userId
    }

    override fun hashCode(): Int {
        return Objects.hash(userId)
    }

}